AssetRipper.GUI/ConsoleOptions.cs

```diff
@@ -0,0 +1,350 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+using System.CommandLine;
+using System.CommandLine.Invocation;
+
+using AssetRipper.Core.IO.MultiFile;
+using AssetRipper.Core.Utils;
+using AssetRipper.Library;
+
+
+namespace AssetRipper.GUI
+{
+	internal enum ExitCode
+	{
+		ExtractingError = 1,
+		ArgumentError = 2,
+	}
```

Comment:
Separate types go in separate files.

Reply:
Sure, it's such a small declaration that I didn't create a new file for it as it'd make it harder to read.